greenmail-core/src/main/java/com/icegreen/greenmail/configuration/PropertiesBasedGreenMailConfigurationBuilder.java

```diff
@@ -80,7 +80,7 @@ public GreenMailConfiguration build(Properties properties) {
         }
 
         String sieveIgnoreDetail = properties.getProperty(GREENMAIL_SIEVE_IGNORE_DETAIL, "false");
-        if (null != sieveIgnoreDetail) {
+        if (Boolean.TRUE.toString().equalsIgnoreCase(sieveIgnoreDetail)) {
```

Comment:
Might be easier to read:
```if (Boolean.parseBoolean(sieveIgnoreDetail)) ...```

Reply:
it is! I just wanted the review to show the details of the actual string comparison taking place but `parseBoolean` is really:
```
public static boolean parseBoolean(String s) {
        return "true".equalsIgnoreCase(s);
    }
```
I hope i will be able to fix later today but if not feel free, and thank you! :)